python/ql/src/semmle/python/Stmts.qll

```diff
@@ -260,6 +281,8 @@ class ExprStmt extends ExprStmt_ {
 /** A for statement, such as `for x in y: print(x)` */
 class For extends For_ {
 
+    /* syntax: for varname in Expr: */
```

Comment:
I'm surprised by the lack of `...` in this and the `async for` statement below.

Reply:
done